### DISCUSSION TITLE
Output of scripts and styles for selected pages.

### DISCUSSION THREAD

**Discussion by kosh2323** [joomla/joomla-cms#41737] [Q&A]
There is such a question, how to connect scripts and styles to Joomla 4, in the <head> section only for those pages where they will be used? For example, there is an image gallery that is displayed on some pages, which can be connected only for selected pages. It is the connection that is interesting, for example, the script is large and it is inconvenient to connect it in the "Debug" position. You need to connect to <head>, and initialize the script on the page itself. How can this be done?

**Comment by drmenzelit** [comment]
You should check the documentation for WebAssetManager. And in the [forum](https://forum.joomla.org) you will find similar questions and the answers too ;-)

**Comment by kosh2323** [comment]
You probably think that whoever is asking questions here does not know where the documentation or the forum is located. I explain to you specifically and to all those who wind up posts here, giving useless and unnecessary advice, this is not in the documentation, the topic has also been created on the forum, and there are no answers yet either. I ask those who know, if there is a specific answer to a specific question, then share this knowledge, and there is no need to flood the topic, giving empty and unverified advice.
